pallene/to_ir.lua

```diff
@@ -87,26 +106,44 @@ function ToIR:exit_function(cmds)
     end
 end
 
---
---
---
-function ToIR:is_local(decl)
-    if decl._tag == "ast.Decl.Decl" then
-        local loc_id = self.loc_id_of_decl[decl]
-        local glb_id = self.glb_id_of_decl[decl]
+-- Returns a to_ir.Var object containing ID of `decl`.
```

Comment:
In addition to returning a Var, this function also has the side effect of registering the upvalues in this function and the surrounding functions. We should mention this in the comment.